### DISCUSSION TITLE
Regarding mdf.select

### DISCUSSION THREAD

**Discussion by JulienGrv** [danielhrisca/asammdf#457] [Ideas]
I had an idea while helping on issue #456 

Instead of:

```python
[...]

​occurrences​ ​=​ ​mdf​.​whereis​(​"my_signal"​)

​signals​ ​=​ ​mdf​.​select​(
    tuple((​None​, ​gp_idx​, ​cn_idx​) ​for​ ​gp_idx​, ​cn_idx​ ​in​ ​occurrences​)
)
```
Would it be possible to support this:

```python
[...]

​occurrences​ ​=​ ​mdf​.​whereis​(​"my_signal"​)

​signals​ ​=​ ​mdf​.​select​(occurrences​)
```

It saves a bit of a headache for users.

**Comment by danielhrisca** [comment]
> ```python
> ​signals​ ​=​ ​mdf​.​get​(occurrences​)
> ```

This looks more like a ``select`` call, probably makes more sense to check if it is a good idea to use that instead

**Reply by JulienGrv** [reply]
Oh! You're right. I need to fix that in the other thread as well.
